admin/src/css/theme.less

```diff
@@ -1,25 +1,47 @@
-.body,
-.header {
-  padding: 20px;
+@nav-active-color: #38464d;
+body {
+  max-width: @admin-content-width;
+  margin: 0 auto;
+  font-size: @font-medium;
 }
 .body {
   display: flex;
 }
+.body>.content {
```

Comment:
We have a `.content` selector a little further below - move this margin rule into the same selector.